srcalc/src/main/java/gov/va/med/srcalc/vista/vistalink/VistaLinkProcedureCaller.java

```diff
@@ -186,7 +186,8 @@ private void setArrayParam(
                 // VistALink represents arrays as newline-delimited strings.
                 if(VlType.array.name().equals(response.getResultsType()))
                 {
-                        // NB: String.split() strips trailing empty strings.
+                    // String.split() is used here because it eliminates an empty string
```

Comment:
is used here... as opposed to? (A first-time reader may not realized that we have standardized on Guava's Splitter.)